rtbkit/common/filter.h

```diff
@@ -434,6 +445,7 @@ struct FilterState
 
     ConfigSet configs_;
     ML::compact_vector<CreativeMatrix, 8> creatives_;
+    FilterReasons filter_reasons_;
```

Comment:
Note that the general convention in this file is to use camel case (as opposed to snake case) for variable names. The underscore for private members was added at the end of the variable name to avoid conflicts with the function names.